op-guide/migration-overview.md

```diff
@@ -1,58 +1,51 @@
 ---
 title: Migration Overview
-summary: Learn how to migrate data from MySQL to TiDB.
+summary: Learn how to migrate data into TiDB.
 category: operations
 ---
 
 # Migration Overview
 
-## Overview
+This document describes scenarios for migrating data into TiDB from either MySQL or another data source via CSV format.
 
-This document describes how to migrate data from MySQL to TiDB in detail.
+## Tools overview
 
-See the following for the assumed MySQL and TiDB server information:
+Migrations will often make use of the following tools. The following is a brief overview of their usage:
 
-|Name|Address|Port|User|Password|
-|----|-------|----|----|--------|
-|MySQL|127.0.0.1|3306|root|* |
-|TiDB|127.0.0.1|4000|root|* |
+- [`mydumper`](../tools/mydumper.md) exports data from MySQL. It is recommended over using mysqldump.
+- [`loader`](../tools/loader.md) imports data in mydumper format into TiDB.
+- [`syncer`](../tools/syncer.md) acts like a MySQL replication slave and pushes data from MySQL into TiDB.
+- [DM](../tools/dm/overview.md) (Data Migration) integrates the functions of mydumper, Loader and syncer to support the export and import of full-size data, as well as incremental synchronization of MySQL Binlog data, and supports data synchronization of a more complete pooled table scenario.
+- [TiDB-Lightning](../tools/lightning/overview-architecture.md) imports data to TiDB in an optimized way. For example, a 1TiB backup could take 24+ hours to import with loader, while it will typically complete at least 3 times faster in TiDB-Lightning.
 
 ## Scenarios
 
-+ To import all the history data. This needs the following tools:
-    - `mydumper`: to export data from MySQL.
-    - `Loader`: to import data to TiDB.
+The following example scenarios show how you can put to use the tools mentioned above.
 
-+ To incrementally synchronize data after all the history data is imported. This needs the following tools:
-    - `mydumper`: to export data from MySQL.
-    - `Loader`: to import data to TiDB.
-    - `Syncer`: to incrementally synchronize data from MySQL to TiDB.
+#### Full data migration from MySQL
 
-        > **Note:** To incrementally synchronize data from MySQL to TiDB, the binary logging (binlog) must be enabled and must use the `row` format in MySQL.
+To migrate the full data, you can use one of the following three solutions:
+  - mydumper + Loader: use mydumper to export data from MySQL and use Loader to import the data into TiDB.
```

Comment:
You may wish to emphasize the **mydumper + Loader:** part of this list.

Reply:
I think we can take another look at formatting after we improve these migration guides. I am going to try and keep it simple, as this overview is a blocker for the DITA TOC.

Thank you for the suggestion though.